.drone.yml

```diff
@@ -77,3 +81,141 @@ volumes:
 - name: socket
   host:
     path: /var/run/docker.sock
+
+---
+kind: pipeline
+name: default-arm64
+
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+- name: fetch
+  image: arm64v8/alpine
```

Comment:
Can we use a different distro, `alpine` uses musl as it's standard libc by default.
Unless most arm oses use musl?

how abou? FROM arm=arm64/ubuntu:18.04

The reason I say mention this is that our go builds aren't completely static, the dns resolver relies on the stdlib at the moment. I will fix that when working on https://github.com/longhorn/longhorn/issues/1768

Reply:
Sorry I don't quite understand your concern here, musl is a library and in arm64 case, it will be compiled for arm64 architecture. I don't think we have to use ubuntu because of this.

Also what do you mean our go builds are not completely static?